.stylelintrc

```diff
@@ -0,0 +1,87 @@
+{
+  "rules": {
```

Comment:
try use 

```
{ 
  "extends": "."
}
```

it should lint itself with the exported rules

Reply:
yep. Done.